frameworks/hdfs/tests/config.py

```diff
@@ -22,7 +22,7 @@
 DEFAULT_HDFS_TIMEOUT = 5 * 60
 HDFS_POD_TYPES = {"journal", "name", "data"}
 KEYTAB = "hdfs.keytab"
-HADOOP_VERSION = "hadoop-2.6.0-cdh5.9.1"
+HADOOP_VERSION = "hadoop-2.6.0-cdh5.16.1"
```

Comment:
Doesn't look like this change belongs here

Reply:
Pushed accidentally, reverted this change to it related branch.